app/controllers/answers_controller.rb

```diff
@@ -1,12 +1,36 @@
 class AnswersController < ApplicationController
+  before_action :find_category
   def index
+
   end
 
   def new
-    @category = Category.find(params[:category_id])
-    @word = @category.words.build
-    @choices = choices.all
+    @words =  @category.words.paginate(page:params[:page], per_page: 1)
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new
   end
 
+  def create
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new(answer_params)
+    if @lesson.save
+      unless params[:page].empty?
+        redirect_to new_category_lesson_answer_url(@word.page: params[:page])
+      else
+        result = 0
+        @cateogyry.words.each do |answer|
```

Comment:
` @cateogyry.words.each do |answer|` what is this?
You are in Lessons Answering, but you are using category?
If you use @category, there will be an error that will say
![image](https://user-images.githubusercontent.com/35232348/105465360-c6e6d600-5ccd-11eb-824e-9eb903e19a75.png)

There are many categories with words, and you are trying to call all the categories with no relationship with ANSWER at all.
What do you think you need? In Lesson Answering, what variable do you need?